Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -92,7 +92,7 @@ If you use a class as a test suite, it must be declared `final`.
 For more information about suites and how to declare and customize them, see
 <doc:OrganizingTests>.
 
-#### Converting setUp() and tearDown() functions
+#### Convert setUp() and tearDown() functions
```

Comment:
Delete; can't have subheadings within a section. Also, don't include symbols in headings.